README.md

```diff
@@ -1,6 +1,11 @@
 [![Build Status](https://travis-ci.com/edazpotato/edazpotato.github.io.svg?branch=master)](https://travis-ci.com/edazpotato/edazpotato.github.io) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e9d6f257fadf43bd884b4435e2480ad8)](https://www.codacy.com/manual/edazpotato/edazpotato.github.io?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=edazpotato/edazpotato.github.io&amp;utm_campaign=Badge_Grade)[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/edazpotato/edazpotato.github.io/blob/master/LICENSE) [![Epicness Badge](https://img.shields.io/badge/epic%3F-yes-brightgreen?logo=Coursera)](http://iamawesome.com/)
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fedazpotato%2Fedazpotato.github.io.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fedazpotato%2Fedazpotato.github.io?ref=badge_shield)
 # edazpotato.github.io #
 Edaz Potato's offcial website!
 
 - Feel free to contribute to this repostory!
 - Lisence: MIT (see [licence](./LICENCE) file)
+
+
+## License
```

Comment:
Codacy found an issue: [[heading-style] Headings should use atx-closed](https://app.codacy.com/manual/edazpotato/edazpotato.github.io/pullRequest?prid=5785031)